### ISSUE TITLE
Declaration files (*.d.ts) is not transformed.

### ISSUE THREAD

**Issue by joshuaavalon** [LeDDGroup/typescript-transform-paths#4]
I create an repository for the bug.

https://github.com/joshuaavalon/typescript-transform-paths-bug

After `npm run build`, `lib\foo\index.js` has

```js
const a_1 = require("..\\a");
```

but `lib\foo\index.d.ts` has

```typescript
import { A } from "@/a";
export declare type B = {
    b: A;
};
```

Edit:

```
{ "transform": "typescript-transform-paths", "afterDeclarations": true }
```

It transforms *.d.ts but it does not tranform .js

**Comment by danielpa9708** [comment]
It seems that 

```
    "plugins": [
      { "transform": "typescript-transform-paths" },
      { "transform": "typescript-transform-paths", "afterDeclarations": true }
    ]
```

works, but I don't know if it is supposed to be like this. We'll have to see if this is an issue with ttypescript or if it the expected behavior.

**Comment by MicahZoltu** [comment]
@danielpza I suspect it is this problem: https://github.com/Zoltu/typescript-transformer-append-js-extension/issues/3#issuecomment-520820776

TL;DR: If anything transforms an import or export declaration, the source visitor will skip it.  By running your transformer `after` or `afterDeclarations`, you end up running after the source visitor, which means you get transformations that the source visitor makes.

**Comment by danielpza** [comment]
Hi @MicahZoltu , uhm, leaving only:

```json
    "plugins": [
      { "transform": "typescript-transform-paths", "after": true }
    ]
```

didn't work for me, the declaration files are not being transformed 

https://github.com/LeDDGroup/typescript-transform-paths/commit/17c149b - test didn't pass

**Comment by MicahZoltu** [comment]
Hmm, in that case this may be a different issue.  I wonder if `after` is before or after `afterDeclarations`?
